specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1733 @@
+      "type": "object",
+      "description": "Aggregated property of each type",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the type."
+        },
+        "count": {
+          "type": "integer",
+          "description": "Total number of items of type."
+        }
+      }
+    },
+    "MonitorServiceList": {
+      "title": "Monitor service details",
+      "description": "Monitor service details",
+      "required": [
+        "data"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/alertsMetaDataProperties"
+        },
+        {
+          "type": "object"
+        }
+      ],
+      "properties": {
+        "data": {
+          "description": "Array of operations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MonitorServiceDetails"
+          }
+        }
+      }
+    },
+    "MonitorServiceDetails": {
+      "description": "Details of a monitor service",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Monitor service name"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Monitor service display name"
+        }
+      }
+    },
+    "alertEnrichmentResponse": {
+      "description": "The alert's enrichments.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/alertEnrichmentProperties"
+        }
+      }
+    },
+    "alertEnrichmentsList": {
+      "description": "List the alert's enrichments.",
+      "properties": {
+        "value": {
+          "description": "List the alert's enrichments",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/alertEnrichmentResponse"
+          }
+        }
+      }
+    },
+    "alertEnrichmentProperties": {
+      "type": "object",
+      "description": "Properties of the alert enrichment item.",
+      "properties": {
+        "alertId": {
+          "type": "string",
+          "description": "Unique Id of the alert for which the enrichments are being retrieved.",
+          "readOnly": true
+        },
+        "enrichments": {
+          "type": "array",
+          "description": "Enrichment details",
+          "items": {
+            "$ref": "#/definitions/alertEnrichmentItem"
+          }
+        }
+      }
+    },
+    "alertEnrichmentItem": {
+      "description": "Alert enrichment item.",
+      "discriminator": "type",
+      "required": [
+        "title",
+        "description",
+        "status",
+        "type"
+      ],
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "The enrichment title."
+        },
+        "description": {
+          "type": "string",
+          "description": "The enrichment description."
+        },
+        "status": {
```

Comment:
>status [](http://example.com/codeflow?start=9&length=6)

is it enumerated?

Reply:
added enum.

Reply:
please add x-ms-enum definition:

"x-ms-enum": {
                "name": <name>,
                "modelAsString": true
              }

Reply:
Hi @noneumar, LGTM except this one. Please fix it.
Please follow  https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process to suppress the errors which are caused by existing APIs in https://github.com/Azure/azure-rest-api-specs/pull/24854/checks?check_run_id=15775032772.
After that, it can be signed off.